parlai/mturk/core/dev/mturk_manager.py

```diff
@@ -1598,22 +1604,37 @@ def get_assignments_for_hit(self, hit_id):
         assignments_info = client.list_assignments_for_hit(HITId=hit_id)
         return assignments_info.get('Assignments', [])
 
+    # FIXME move this into mturk_utils
+    def check_hit_status(self, hit_id):
```

Comment:
just a note that the side effect of removing the hit from the hit_id list is a little ambiguous given the function name (I figured this was happening based on code above but was not sure until examining the function)

Reply:
Yeah this is somewhat unclear from the name. Picking a better one.